### ISSUE TITLE
后训练和推理的时候 action_type, action_mode 到底用 delta 还是 abs?

### ISSUE THREAD

**Issue by Droliven** [InternRobotics/InternVLA-A1#24]
- tutorials/finetune_internvla_a1_5_with_robotwin.md 这个后训练脚本里面，ACTION_TYPE=delta
- evaluation/RoboTwin/eval.sh 默认的 ACTION_MODE=abs

所以robotwin后训练的模型到底用什么mode?

**Comment by mahaoxiang822** [comment]
If you choose to fine-tune based on the pre-trained base model, both `delta` and `abs` action modes will work fine. 

However, if you plan to use the fine-tuned RoboTwin model directly from Hugging Face, please note that the action type for that specific model is `abs`.

**Comment by ChengAnda** [comment]
What would happen if the **abs** mode were used throughout the pre-training process?

**Comment by mahaoxiang822** [comment]
> What would happen if the **abs** mode were used throughout the pre-training process?

Because the action definition of different robots can be different, using 'abs' action can result very sparse label space and maybe downgrade the pretrain performance
